platform/solr/solr-appender/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,34 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+    <OCD description="Decanter Solr Appender"
+         name="Solr Appender" id="org.codice.solr.appender.SolrAppender">
+
+        <AD description="The maximum batch size for solr writes."
```

Comment:
❓ These are fairly advanced configurations that I doubt admins will ever need to change. Should we just use undocumented system properties for now instead of exposing those in the configuration admin? @pklinef, thoughts?